history/characters/200000_farraki.txt

```diff
@@ -8632,3 +8634,57 @@
 	635.8.7={ birth=yes }
 	718.9.14={ death=yes }
 }
+ruuzlu = { # right-hand man and friend of Ukorz Sandscalp
+	name = Ruuzlu
+	culture=farraki religion=farrism
+	martial=8 diplomacy=6 stewardship=5 intrigue=9 learning=4
+	trait=education_intrigue_3 trait=diligent trait=sadistic trait=deceitful
+	586.1.30={ birth=yes }
+	592.08.13={
+		employer = 200104 # Ukorz
+	}
+	596.1.30={
+		set_relation_friend = character:200104 # Ukorz Sandscalp
+	}
+	653.9.30={ death=yes }
+}
+sul = { # member of the Council of Elders
+	name = Sul
+	culture=farraki religion=farrism
+	martial=7 diplomacy=5 stewardship=5 intrigue=8 learning=7
+	trait=education_intrigue_3 trait=arrogant trait=ambitious trait=diligent
+	538.8.29={ birth=yes }
+	563.8.29={
+		give_nickname = nick_the_sandcrawler
+	}
+	613.7.5={ # slain by adventurers in Throne of Thunder
+		death = {
+			death_reason = battle
+		}
+	}
+}
+jakrazet = {
+	name = Jakra'zet
+	culture = farraki
+	religion = farrism
+	martial=9 diplomacy=5 stewardship=5 intrigue=8 learning=5
+	trait=education_martial_4 # his strategic thinking earned him the position of General
+	trait=desert_warrior
+	trait=diligent trait=deceitful trait=zealous
```

Comment:
Looks like his main drive was revenge on the Horde for the fall of his tribe.
Especially considering the fact he converted to the G'Huun heresy.
```suggestion
	trait=diligent trait=deceitful trait=vengeful
```